hypervisor/common/sched_bvt.c

```diff
@@ -8,17 +8,70 @@
 #include <per_cpu.h>
 #include <schedule.h>
 
-int sched_bvt_init(__unused struct sched_control *ctl)
+#define CONFIG_MCU_MS	1U
+#define CONFIG_CSA_MCU_NUM 5U
+struct sched_bvt_data {
```

Comment:
Moving the definition into the first patch is better?

Reply:
ok